code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -40,7 +40,7 @@
 		return
 	has_suit.overlays -= inv_overlay
 	has_suit = null
-	src.loc = get_turf(user || src)
+	src.forceMove(get_turf(user || src))
```

Comment:
Read the rest of it.

"The first true value from left to right completes the evaluation (a practice known as short-circuiting). **The entire expression takes the value of the last argument to be evaluated.**"